pywwise/ak/wwise/core/transport.py

```diff
@@ -1,54 +1,154 @@
+        sure you create a transport object before calling this function.
+        :return: True if the transport object was destroyed, False otherwise.
+        """
+        if transport_id is None:
+            return False
+        
+        args = {"transport": transport_id}
+        
+        return self._client.call("ak.wwise.core.transport.destroy") is not None
+    
+    def execute_action(self, transport_action: ETransportExecuteActions, transport_id: int = None) -> bool:
+        """
+        https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_transport_executeaction.html \n
+        Executes an action on the given transport object, or all transport objects if none is specified.
+        :arg transport_action: The action to execute. Use ETransportExecuteActions to see the available actions.
+        :arg transport_id: The transport object ID to be used with all other ak.wwise.core.transport functions. Make
+        sure you create a transport object before calling this function.
+        :return: True if the action was executed, False otherwise.
+        """
+        if transport_action is None:
+            return False
+        
+        args = {"action": transport_action, **({"transport": transport_id} if transport_id is not None else {})}
+        
+        return self._client.call("ak.wwise.core.transport.execute", args) is not None
+
+    def get_list(self) -> tuple[TransportObjectInfo, ...]:
+        """
+        https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_transport_getlist.html \n
+        Returns the list of transport objects.
+        :return: A list of transport objects.
+        """
+        results = self._client.call("ak.wwise.core.transport.getList", {})
+        
+        results = results.get("list") if results is not None else None
+        
+        if results is None:
+            return ()
+        
+        transport_objects: list[TransportObjectInfo]
+        
+        for result in results:
+            object = GUID(result["object"])
+            game_object = int(result["gameObject"])
+            transport = int(result["transport"])
+            transport_objects.append(TransportObjectInfo(object, game_object, transport))
+        
+        return tuple(transport_objects)
+    
+    def get_state(self, transport_id: int) -> ETransportState:
+        """
+        https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_transport_getstate.html \n
+        Gets the state of the given transport object.
+        :param transport_id: The transport object ID to be used with all other ak.wwise.core.transport functions. Make
+        sure you create a transport object before calling this function.
+        :return: An enum value that represents the state of the transport object. Return value can either be: Playing,
+        Stopped, Paused, or None. In case its none, the method call failed.
```

Comment:
New lines in `:param:` and `:return:` are misaligned. Can cause docstring rendering issues.